index.bs

```diff
@@ -995,6 +999,17 @@ When this method is invoked, the user agent MUST run the following steps:
 
 </div>
 
+
+<div class="algorithm" data-algorithm="apply-frame-updates">
+
+To <dfn for=XRFrame>apply frame updates</dfn> for an {{XRFrame}}, the user agent MUST run the following steps:
+
+ 1. Do nothing.
+
+</div>
+
+NOTE: Further specs may add new kinds of frame updates to these steps, e.g. for gamepad state.
```

Comment:
This seems somewhat odd to me, but I'm not sure what else to suggest.  Is there someone with experience with this sort of problem that we could get a sanity check from?

Reply:
An alternate plan was to have a "list of frame updates" (a list of callbacks) on the session and session with gamepads append a frame update to it. I felt that might get a bit unweildy though.